resources/js/processes/modeler/components/inspector/ScreenSelect.vue

```diff
@@ -23,6 +23,14 @@
         </multiselect>
         <small v-if="error" class="text-danger">{{ error }}</small>
         <small v-if="helper" class="form-text text-muted">{{ $t(helper) }}</small>
+        <a
+                v-if="content.id"
+                :href="`/designer/screens/${content.id}/edit`"
+                target="_blank"
+        >
+            Open Screen
```

Comment:
Are translations available for this link text? Could it be wrapped in `$t`?

Reply:
No translations. Should I wrap it?

Reply:
"Open Process" isn't wrapped. So I followed that.

Reply:
I should have wrapped "Open Process". I think it makes sense to, as everything seems to be translated. Worst case is the translation doesn't exist and it shows the text as-is.